app/grants/clr.py

```diff
@@ -384,6 +407,38 @@ def populate_data_for_clr(grants, contributions, phantom_funding_profiles, clr_r
 
 
 
+def analytics_clr(from_date=None, clr_round=None, network='mainnet'):
```

Comment:
Do we wanna move this into it's own file ?

Reply:
I just left it there because if we moved it into its own file...I'm assuming we would have to port over alot of the same methods used in the `clr.py` file, no?

Reply:
Oh you can still import those methods into your analytics file !
If you need xyz method
import xyz from app/grants/clr.py sort of thing

Reply:
done. So, maybe a stupid question. How come in `analytics_clr.py` I have to import ` fetch_data`, `populate_data_for_clr`, and `calculate_clr_for_donation` but you don't have to do so in `estimate_clr.py`, even though `predict_clr` uses the same methods?

Reply:
Oh no you wouldn't have to do that 🙌